<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QWidget" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>686</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(206,239,228,60%);</string>
  </property>
  <widget class="QPushButton" name="pbt_addfriend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 0%);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="Label_user">
      <property name="text">
       <string>当前用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Label_UserName">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="friend_list">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>230</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="friend_message">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 0%);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_friendask">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QListWidget" name="chatwidget">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>10</y>
     <width>391</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_sendmess">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>600</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend_chatname">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 0%);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="unline_friend_list">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="unline_cnt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 0%);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="inline_cnt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 0%);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="friend_flag_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>201</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SendEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>600</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LocalFindFriend">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
  </widget>
  <widget class="QLabel" name="find_inlineno">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="find_unlineno">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, 40%);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconlabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>-10</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/xtu_mag.png);
background-color: rgba(255,255,255, 0%);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
